### ISSUE TITLE
Courses Listed As Downloading But Then  Stops

### ISSUE THREAD

**Issue by ThePieMonster** [yoonwaiyan/stackskills-dl#36]
I can login and it will show the list of courses but the script ends right after that. Doesn't seem like it even made a attempt to begin downloading.

System: Windows 10
Ruby: ruby 2.4.4p296 (2018-03-28 revision 63013) [x64-mingw32]

> WARN: Unresolved specs during Gem::Specification.reset:
      unf_ext (>= 0)
WARN: Clearing out unresolved specs.
Please report a bug if this causes problems.
Login Email: 
Login password: 
Login Successfully.
Number of courses found: 1
Downloading Course: CourseName
Downloading 00a._CourseName_1
Downloading 00b._CourseName_2
Downloading 00b._CourseName_etc...

**Comment by yoonwaiyan** [comment]
Hi, may I know what's inside of each courses e.g. is that a Vimeo video, or a download link of the video or any other resources such as text files or zip files?

**Comment by ThePieMonster** [comment]
> Hi, may I know what's inside of each courses e.g. is that a Vimeo video, or a download link of the video or any other resources such as text files or zip files?

Here is the course id
https://stackskills.com/courses/enrolled/182238

**Comment by yoonwaiyan** [comment]
Sorry I couldn't access to the course as I'm not enrolled and it's not free.

**Comment by ThePieMonster** [comment]
> Sorry I couldn't access to the course as I'm not enrolled and it's not free.

From the icons displayed to the left of the course title it just shows a "YouTube" styled video icon. I clicked threw a bunch of them and they should all only be videos.

I wanted to add that I tried this in a Ubuntu VM and it is working as expected. Seems to be a Windows issue. Let me know if there's anything you would like me to try to get this working on Windows as well.

**Comment by yoonwaiyan** [comment]
Seems to be related to the ol' wget problems as mentioned few times on the issues being reported. Would appreciate if you could verify this but it's ok too if it's troublesome to you. I have a plan to Dockerize this repo to make it platform independent and setup-free.

**Comment by ThePieMonster** [comment]
> Seems to be related to the ol' wget problems as mentioned few times on the issues being reported. Would appreciate if you could verify this but it's ok too if it's troublesome to you. I have a plan to Dockerize this repo to make it platform independent and setup-free.

Well I went threw the same process on Windows as I did on Linux for testing the Ruby script so I'd say it is that issue. I also was testing the Docker version of this just now and I am getting the below error. In docker-compose.yml my email and password is set, saved, and re-ran the build command but still get the below error.

`PS C:\git\stackskills-dl> docker-compose run stackskills_dl`
`WARNING: The PWD variable is not set. Defaulting to a blank string.`
`ERROR: Cannot create container for service stackskills_dl: b'Mount denied:\nThe source path "\\\\downloads:/usr/app/downloads"\nis not a valid Windows path'`

Fixed the Error by running the below command in a PowerShell window. Recommend adding this to your ReadMe in a troubleshooting section or something. 👍 
`$Env:COMPOSE_CONVERT_WINDOWS_PATHS=1`
